internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -182,7 +182,7 @@ func (r *ClusterDeploymentsReconciler) Reconcile(origCtx context.Context, req ct
 
 	cluster, err := r.Installer.GetClusterByKubeKey(req.NamespacedName)
 	if errors.Is(err, gorm.ErrRecordNotFound) {
-		if !isInstalled(clusterDeployment, clusterInstall) {
+		if !clusterDeployment.Spec.Installed {
```

Comment:
@rollandf Pay attention to this one

Reply:
LGTM

Reply:
Why the change is needed?

Reply:
If I create a new ClusterDeployment with installed==true, it will currently create a day1 cluster because the conditions aren't set

Reply:
I must miss something. First thing checked in `isInstalled` is if `clusterDeployment.Spec.Installed`
```
func isInstalled(clusterDeployment *hivev1.ClusterDeployment, clusterInstall *hiveext.AgentClusterInstall) bool {
	if clusterDeployment.Spec.Installed {
		return true
	}
	cond := FindStatusCondition(clusterInstall.Status.Conditions, hiveext.ClusterCompletedCondition)
	return cond != nil && cond.Reason == hiveext.ClusterInstalledReason
}
```

Reply:
You're right, my bad, misread the code